libs/designer/src/lib/ui/panel/panelTabs/parametersTab/index.tsx

```diff
@@ -136,8 +136,8 @@ const ParameterSection = ({
       const { viewModel } = newState;
       const parameter = nodeInputs.parameterGroups[group.id].parameters.find((param: any) => param.id === id);
       if (
-        (parameter?.type === constants.SWAGGER.TYPE.BOOLEAN && value.length === 1 && value[0]?.value === 'True') ||
-        value[0]?.value === 'False'
+        (parameter?.type === constants.SWAGGER.TYPE.BOOLEAN && value.length === 1 && value[0]?.value?.toLowerCase() === 'true') ||
+        value[0]?.value?.toLowerCase() === 'false'
```

Comment:
this too, remove

Reply:
This is needed for #2012

Reply:
Still separate PR, this PR is fixing two other bugs which is not convernient if we want to revert